css/subscribe.scss

```diff
@@ -1,3 +1,6 @@
+---
+---
```

Comment:
Thanks for contributing, @rohitsrma. 

Are these two lines intentional?

Reply:
Yes, it tells Jekyll to process the file.

https://jekyllrb.com/docs/step-by-step/07-assets/